docs/user/visualize.asciidoc

```diff
@@ -7,74 +7,79 @@ _Visualize_ enables you to create visualizations of the data from your {es} indi
 
 {kib} visualizations are based on {es} queries. By using a series of {es} {ref}/search-aggregations.html[aggregations] to extract and process your data, you can create charts that show you the trends, spikes, and dips you need to know about.
 
+To begin, open *Visualize*, then click *Create new visualization*.
+
 [float]
-[[create-a-visualization]]
-== Create visualizations
+[[visualization-types]]
+== Types of visualizations
+
+To help you choose the best visualization for your data, {kib} supports many types of visualizations.
 
-. Open *Visualize*.
-. Click *Create new visualization*.
-. Choose the visualization type:
-+
-* Basic charts
-[horizontal]
 <<lens,Lens>>::
 Quickly build several types of basic visualizations by simply dragging and dropping the data fields you want to display.
 
-* *<<most-frequent,Most frequently used visualizations>>*
-[horizontal]
-Line&comma; area, and bar charts:: Compare different series in X/Y charts.
+<<most-frequent,Most frequently used visualizations>>::
```

Comment:
Remove "Most"

Reply:
If I change this instance, I'll have to change it everywhere. I'll do this in another PR.